src/app/reviewer/[orgSlug]/study/[studyId]/review/study-review-buttons.tsx

```diff
@@ -44,9 +51,15 @@ export const StudyReviewButtons: FC<{ study: SelectedStudy }> = ({ study }) => {
 
     const TestingCheck = () => {
```

Comment:
instead of defining this inline in the component, lets move it to a component outside that returns null if the user isn't an admin